genome.spec

```diff
@@ -1,34 +1,45 @@
 /*
-KBase genome ID
+Workspace reference to a genome object.
 @id kb
 */
 typedef string Genome_id;
 
 /*
-Reference to a source_id
+External database reference from which this genome data originated.
+Examples: "NC_014248"
 @id external
```

Comment:
FYI `@id`s where the target is anything but `ws`, `handle`, or `bytestream` don't do anything